File: Techschool/Techschool/ClientApp/src/app/models/vacations/social-with-children-vacation.model.ts
import { VacationModel } from './vacation.model';

export class SocialWithChildrenVacationModel extends VacationModel {
  orderNumber: string;
  orderDate: Date;

  constructor (data: SocialWithChildrenVacationModel = null) {
    super();
    if (data) {
      this.id = data.id;
      this.startOfVacationDate = new Date(data.startOfVacationDate);
      this.endOfVacationDate = new Date(data.endOfVacationDate);
      this.orderNumber = data.orderNumber;
      this.orderDate = new Date(data.orderDate);
      this.workingYearId = data.workingYearId;
    }
  }
}

File: Techschool/Techschool.BLL/Models/Vacations/SocialWithChildrenVacationModel.cs
﻿using System;

namespace Techschool.BLL.Models.Vacations
{
    public class SocialWithChildrenVacationModel : VacationModel
    {
        public DateTime StartOfVacationDate { get; set; }
        public DateTime EndOfVacationDate { get; set; }
        public string OrderNumber { get; set; }
        public DateTime OrderDate { get; set; }
        public string SocialWithChildrenVacationFormId { get; set; }
    }
}
